### PR TITLE
test(sizes): add edge case tests

### PR DESCRIPTION
**Pull request by luqven**
# Description
This PR removes deprecated fn arguments and adds edge-case tests to the cypress tests. It also moves the cypress html fixture's css into it's own file.

### CODE REVIEW

File: src/autoSize.js [hunk 1]
```diff
@@ -36,15 +36,19 @@ const imageLoaded = ({ el }) => {
   // weren’t downloaded as not complete. Some Gecko-based browsers
   // report this incorrectly. More here: https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-complete
   if (!el.complete) {
-    console.error('not complete');
+    console.warn(
+      'Imgix.js: attempted to set sizes attribute on element with complete evaluating to false'
+    );
```
Comment (frederickfogerty) [inline src/autoSize.js:41]: I like these copy changes 👏

**Review by frederickfogerty — APPROVED** [review]
Nice 👏

**Review by sherwinski — APPROVED** [review]
